nautobot/docs/user-guide/platform-functionality/echarts.md

```diff
@@ -0,0 +1,210 @@
+        description="Example chart",
+        data={
+            "Compliant": {"aaa": 5, "dns": 12, "ntp": 8},
+            "Non Compliant": {"aaa": 10, "dns": 20, "ntp": 15},
+        },
+    )
+
+    chart_config = chart.get_config()
+
+    return {
+        "chart": chart,
+        "chart_config": chart.get_config(),
+        "chart_width": self.width,
+        "chart_height": self.height,
+        "chart_container_id": slugify(f"echart-{self.header}"),
+    }
+```
+
+Corresponding Template (`detail.html`)
+
+```no-highlight
+{% load helpers %}
+{% render_echart chart chart_config chart_widht chart_height chart_container_id %}
+```
+
+## Example Usage as a Nautobot UI Component
+
+### Bar Type
+
+```no-highlight
+from nautobot.core.ui import object_detail
+from nautobot.core.ui.choices import EChartsTypeChoices, SectionChoices
+
+object_detail_content = object_detail.ObjectDetailContent(
+    panels=[
+        object_detail.EChartsPanel(
+            section=object_detail.SectionChoices.FULL_WIDTH,
+            weight=100,
+            label="EChart - BAR",
+            chart_kwargs={
+                "chart_type": EChartsTypeChoices.BAR,
+                "header": "Traffic per Interface Bar",
+                "description": "Example bar chart from EChartsBase",
+                "data": {"Compliant": {"aaa": 5, "dns": 12, "ntp": 8}, "Non Compliant": {"aaa": 10, "dns": 20, "ntp": 15}},
+            },
+        ),
+    ]
+)
+```
+
+![Bar echarts data](../../media/development/echarts/bar-echarts-data.png)
+
+### Pie Type
+
+```no-highlight
+from nautobot.core.ui import object_detail
+from nautobot.core.ui.choices import EChartsTypeChoices, SectionChoices
+
+object_detail_content = object_detail.ObjectDetailContent(
+    panels=[
+        object_detail.EChartsPanel(
+            section=object_detail.SectionChoices.LEFT_HALF,
+            weight=100,
+            label="EChart - PIE",
+            chart_kwargs={
+                "chart_type": EChartsTypeChoices.PIE,
+                "header": "Traffic per Interface Pie",
+                "description": "Example pie chart from EChartsBase",
+                "data": {"Traffic Sources": {"Direct": 335, "Email": 310, "Union Ads": 234, "Video Ads": 135}},
+            },
+        ),
+    ]
+)
+```
+
+<!-- pyml disable-num-lines 3 no-inline-html -->
+<p align="center">
+    <img src="../../media/development/echarts/pie-echarts-data.png"/>
+</p>
+
+### Line Type
+
+```no-highlight
+from nautobot.core.ui import object_detail
+from nautobot.core.ui.choices import EChartsTypeChoices, SectionChoices
+
+object_detail_content = object_detail.ObjectDetailContent(
+    panels=[
+        object_detail.EChartsPanel(
+            section=object_detail.SectionChoices.RIGHT_HALF,
+            weight=100,
+            label="EChart - LINE",
+            chart_kwargs={
+                "chart_type": EChartsTypeChoices.LINE,
+                "header": "Traffic per Interface Line",
```

Comment:
Can you update this to reflect the data type, and update the image to go with it.